blog/docs/articles/performance-test-with-slos.md

```diff
@@ -0,0 +1,183 @@
+# Dead Simple Performance Testing and SLOs for Knative Apps
+
+**Author: [Srinivasan Parthasarathy](https://researcher.watson.ibm.com/researcher/view.php?person=us-spartha), Senior Research Scientist and Manager, DevSecOps @ [IBM Research](https://research.ibm.com)**
+
+**Date: 2022-07-18**
+
+
+> Launch performance tests for Knative services (HTTP and gRPC) and validate their service-level objectives (SLOs) **in seconds**.
+
+Performance testing is a core building block in the robust delivery of HTTP and gRPC services. One way to accomplish this is by sending a stream of requests to the target service, and evaluating the responses for error and latency-related violations. From a developer’s perspective, this approach has three main considerations, namely, i) the load-related characteristics of the request stream, such as the request rate; ii) the shape of the requests, in particular, whether the service requires any payload/data to be sent as part of the requests; and iii) the service-level objectives (SLOs) used to validate the quality of the target service.
+
+You can use [Iter8](https://iter8.tools), the open source Kubernetes release optimizer, to flexibly launch performance tests for Knative services **in seconds**, with precise control over all of the above. This article introduces these capabilities.
+
+## Overview of Iter8
+
+Iter8 is the Kubernetes release optimizer built for DevOps, MLOps, SRE and data science teams. Iter8 makes it easy to ensure that Kubernetes apps and ML models perform well and maximize business value.
+
+Iter8 supports the following use-cases.
+
+1.  Performance testing and SLO validation of HTTP services.
+2.  Performance testing and SLO validation of gRPC services.
+3.  SLO validation using custom metrics from any database(s) or REST API(s).
+
+### Iter8 experiment
+Iter8 introduces the notion of an [experiment](https://iter8.tools/0.11/getting-started/concepts/#iter8-experiment), which is a list of configurable tasks that are executed in a specific sequence.
+
+<img src="/blog/docs/articles/images/iter8-intro-dark.png" alt="Iter8 experiment" width="600"/>
```

Comment:
This image is not displaying

<img width="474" alt="image" src="https://user-images.githubusercontent.com/44698588/179822449-bd69a97a-63e4-4b7c-8f03-e99c4bb07aec.png">

Reply:
(think it may be the beginning `/` but not sure...

Reply:
I placed the images in the wrong subfolder. Fixed now. Hopefully, should work. 

***

On a related note, it would be nice if there's a way for me to do `mkdocs serve -s` and simply navigate the blog articles myself ... this way, I can self check that things are working fine before I push the PR.

I tried doing this but my command exited with an error:

```shell
cd blog
mkdocs serve -s
```

Above yields:

```shell
INFO     -  Building documentation...
ERROR    -  Config value: 'theme'. Error: The path set in theme.custom_dir ('/Users/srinivasanparthasarathy/Documents/Projects/docs-1/blog/overrides') does not exist.
Aborted with 1 Configuration Errors!
```

If blog authors can spin up mkdocs locally, it can be really helpful in article creation.

Reply:
You can do this by running `BUILD_VERSIONS=no ./hack/build.sh serve` from the root directory

Reply:
`<img src="../images/iter8-intro-dark.png" alt="Iter8 experiment" width="600"/>` should work

Reply:
I believe the pics should work now. Also improved the formatting significantly.

Reply:
Fixed.